test/python/unittest/Test.py

```diff
@@ -0,0 +1,28 @@
+""" :module: Top level test suite  """
```

Comment:
please add the same copyright line as in the other file :)

Reply:
done